newIDE/app/src/GameDashboard/GameCard.js

```diff
@@ -43,48 +43,52 @@ export const GameCard = ({
         subheader={
           <Line alignItems="center" noMargin>
             <Trans>
-              Registered on{' '}
+              Created on{' '}
               {format(game.createdAt * 1000 /* TODO */, 'yyyy-MM-dd')}
```

Comment:
You could format the date with https://github.com/4ian/GDevelop/blob/master/newIDE/app/src/Profile/Achievement/AchievementList.js#L125